### ISSUE TITLE
bf command line tools silently fail

### ISSUE THREAD

**Issue by tomlm** [microsoft/botframework-cli#1005]
## Versions
4.10, 4.11

## Describe the bug
When you use bf cli as part of a script when it fails it doesn't set the process error code which means that it silently fails.  Our lu files stopped building because of some internal error and for the last month we have not updated our models because the silent failure.  

## To Reproduce
create a cmd file like this:
```
call bf luis:build --foo
if errorlevel 1 @echo error 
if not errorlevel 1 @echo success
```

## Expected behavior
It should say "error" and it says "success"

[bug]

**Comment by feich-ms** [comment]
That make sense, looking.

**Comment by feich-ms** [comment]
fixed by #1022
